src/components/home/header/swiper/swiper.tsx

```diff
@@ -20,6 +20,57 @@ import properties from "assets/images/suggest/properties.png";
 import fashion from "assets/images/suggest/fashion.png";
 
 export default function SwiperContainer() {
+  let imgTitleMap = [
```

Comment:
1.You can move the array to separate file.
2. You use props to clean that iteration to separate component.
```
<ImageSlider images={images}/>
```